templates/verify.txt

```diff
@@ -1,6 +1,6 @@
-{{t "Almost there"}}
+{{t "Welcome!"}}
```

Comment:
@ryanfeeley mailchip warns against "Excessive use of exclamation points" http://mailchimp.com/resources/guides/how-to-avoid-spam-filters/html/

Since the exclamation point here doesn't really add value, it might be better to err on the side of caution and get rid of it.